### PR TITLE
Contact Form

### PR DESCRIPTION
**Pull request by Zainab6**
the form is done by MDBreact with title and icon for send
I add the translation also
Annotation 2020-08-11 172346

### CODE REVIEW

File: src/components/ContactForm/contactForm.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                    group
+                    type="email"
+                    validate
+                    error="wrong"
+                    success="right"
+                  />
+                </MDBCol>
+              </MDBRow>
+              <MDBInput
+                label={t('ContactUsPage.email')}
```
Comment (QaysAyad) [inline src/components/ContactForm/contactForm.js:63]: I thing this should be message
```suggestion
                label={t('ContactUsPage.message')}
```

File: src/components/ContactForm/contactForm.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                validate
+                error="wrong"
+                success="right"
+              />
+              <MDBInput
+                type="textarea"
+                rows="2"
+                style={{ width: '550px' }}
+                label={t('ContactUsPage.message')}
+                icon="paper-plane"
```
Comment (QaysAyad) [inline src/components/ContactForm/contactForm.js:75]: I can see that the icon inside the testarea and that is fine but i believe it is not clickable make sure that the icon is a button and it is clickable

File: src/components/ContactForm/contactForm.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import '../../assets/main.css';
```
Comment (QaysAyad) [inline src/components/ContactForm/contactForm.js:2]: You don't need to import this it is already imported in the index.js so it going to apply to the whole app anyway just delete this line

File: public/locales/en/translation.json [hunk 4]
```diff
@@ -1 +1,9 @@
-{}
+{
+  "ContactUsPage": {
```
Comment (QaysAyad) [inline public/locales/en/translation.json:2]: Use the same camelcase as for variables the first letter is small and every other first letter is capital you have done it right with "firstName" and "lastName" 
 ```suggestion
  "contactUsPage": {
```

File: src/components/ContactForm/contactForm.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                group
+                type="text"
+                validate
+                error="wrong"
+                success="right"
+              />
+              <MDBInput
+                type="textarea"
+                rows="2"
+                style={{ width: '550px' }}
```
Comment (MohammedAlabd) [inline src/components/ContactForm/contactForm.js:73]: try to remove the inline styling

File: src/components/ContactForm/contactForm.js (hunk 1 above)
Comment (Zainab6) [reply src/components/ContactForm/contactForm.js:63]: the email is the 3rd field while message is the 4th

**Pull request merged**